File: programming-concepts/input-output/python/double_entry.py
# Raspberry Pi Foundation
# Developed as part of Ada Computer Science
# 
# Usage licensed under CC BY-NC-SA 4.0

def main():
    email = input("Enter your email address: ")
    verify_email = input("Re-type your email address: ")

    if email == verify_email:
        print("The emails match")
    else:
        print("Error: the email addresses did not match")


# This code will run if this file is executed directly
# (i.e. not called by another program)
if __name__ == '__main__':
    main()

File: programming-concepts/input-output/c/double_entry.c
#include <stdio.h>
#include <string.h>


int main()
{
    //email address and email verification
    char email[250];
    char email2[250];
    printf("Enter your email address: ");
    scanf("%s", email);
    printf("Enter your email address again: ");
    scanf("%s", email2);
    if (strcmp(email, email2) == 0)
    {
        printf("Emails match\n");
    }
    else
    {
        printf("Emails do not match\n");
    }
    return 0;
}